frontends/main/src/page-components/SearchDisplay/SearchDisplay.tsx

```diff
@@ -230,13 +230,22 @@ const FacetStyles = styled.div`
       .input-postfix-button {
         cursor: pointer;
         position: absolute;
-        right: 5px;
-        top: 50%;
-        transform: translateY(-50%);
+        right: 7px;
+        top: 9px;
```

Comment:
This looks/works fine, so 👍 

However, pixels for vertical centering always seem fragile to me. The reason this wasn't working as expected before is shown below

<img width="296" alt="Screenshot 2024-10-10 at 10 38 14 AM" src="https://github.com/user-attachments/assets/c52abd70-11eb-4c08-ae2b-e317271fb0d5"> <img width="396" alt="Screenshot 2024-10-10 at 10 40 33 AM" src="https://github.com/user-attachments/assets/e19ec31a-9075-4865-ad57-640bc0d9a1ab">

The issues were:
- `input` had margin, which affected the content-box height of wrapper. If the margin goes on the wrapper instead, then `top: 50%; transform: translate(-50%)` works as expected *for centering the button*
- The button is slightly larger than the SVG.
    - This seems to be because the SVG is `display: inline`. If you change it to block, the button is same height as SVG. 
    - I don't completely understand how the height of a container with inline contents is computed. If you know or can point me to a good reference, I'd love to understand better. 
        - I was trying to understand this the other day and ended up here: https://stackoverflow.com/questions/11126685/why-does-container-div-insist-on-being-slightly-larger-than-img-or-svg-content but that didn't really tell me how the container height is calculated, just that it's "different".  

**Anyway:** If you want to use the percentage approach, (1) put margin on wrapper, (2) make sure button is same size as SVG.

Reply:
I've pushed the change to center at 50% - I agree the positioning shouldn't be susceptible to anything that might change the container height.

Also, the facet heading were not centered, for the same reason you pointed out above, in this case the chevron svg was displayed inline. It will be due to `line-height` being accounted for for inline elements, though as always with css the exact rules are byzantine to me!